builtin/providers/aws/resource_aws_iam_server_certificate.go

```diff
@@ -132,9 +140,10 @@ func resourceAwsIAMServerCertificateDelete(d *schema.ResourceData, meta interfac
 }
 
 func normalizeCert(cert interface{}) string {
-	if cert == nil {
+	if cert == nil || cert == (*string)(nil) {
```

Comment:
This doesn't seem like it should be necessary?

https://play.golang.org/p/gED99KcQnP